src/tutorial/scheduling/datetime-cycling.rst

```diff
@@ -637,3 +686,45 @@ Putting It All Together
          +          +PT12H/PT6H = """
          +              forecast[-PT6H] => forecast
          +          """
+
+   #. **Clock Triggers**
+
+      To ensure that the ``get_observations_<location>`` tasks run only
+      after the time of the observation add a clock trigger.
+      Observations are available by 10 minutes past the hour.
+      We don't want to overburden the Cylc server, so check for observations
+      every 5 minutes.
+
+      .. spoiler:: Hint hint
+
+         See :ref:`tutorial-clock-triggers`
+
+      .. spoiler:: Solution warning
+
+         .. note::
+
+            There are a number of ways of writing the dependency: This
+            example shows multiple patterns.
```

Comment:
It's fine to mention there are multiple ways of doing this, but I would only show 1 way for consistency.